internal/services/oracle/oracle_cloud_vm_cluster_resource.go

```diff
@@ -292,8 +292,10 @@ func (CloudVmClusterResource) Arguments() map[string]*pluginsdk.Schema {
 		"tags": commonschema.Tags(),
 
 		"file_system_configuration": {
-			Type:     pluginsdk.TypeList,
+			Type: pluginsdk.TypeList,
+			// O+C - the value will be different between creation and reading because the API provides an additional immutable or default elements
 			Optional: true,
+			Computed: true,
```

Comment:
Can we update this to follow the HashiCorp standards for [O+C fields](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/best-practices.md#setting-properties-to-optional--computed)
```suggestion
			Type: pluginsdk.TypeList,
			Optional: true,
			// O+C - the value will be different between creation and reading because the API provides an additional immutable or default elements
			Computed: true,
```